docs/guides/add-exposure-tests.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Add exposure validation tests"
+---
+
+After you [install the dbt package](/quickstart#install-the-dbt-package), you can add Elementary exposure validation tests.
+
+## Exposure validation dbt tests
+
+Elementary dbt package includes **exposure validation tests, implemented as [dbt tests](https://docs.getdbt.com/docs/building-a-dbt-project/tests)**.
+These tests can detect missing columns and column data types for defined exposures.
```

Comment:
@erikzaadi 
This is a technical description of the test. 
I think it's better to describe here the use case it solves. 
"These tests will fail if there are column changes that break downstream exposures, such as BI dashboards."

Also, it's a good place to say that Elementary Cloud automatically generates exposures, maps the lineage, and adds the test to the models that the exposures depend on.